roles/commcare_analytics/templates/postgres/postgresql.conf.j2

```diff
@@ -66,7 +66,7 @@ listen_addresses = 'localhost' # what IP address(es) to listen on;
 					             # defaults to 'localhost'; use '*' for all
 					             # (change requires restart)
 port = 5432				# (change requires restart)
-max_connections = 100			# (change requires restart)
+max_connections = 200			# (change requires restart)
```

Comment:
I am not sure how much this number could rise to. I will get to know when i test things on staging.